babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -788,12 +788,64 @@ SqlIndex SqlCreateTableIndex(Span s) :
 SqlNode SqlExecMacro() :
 {
     SqlIdentifier macro;
+    SqlNodeList params = new SqlNodeList(getPos());
     Span s;
 }
 {
     macro = CompoundIdentifier() { s = span(); }
+    [
+        SqlExecMacroArgument(params)
+    ]
+    {
+        return new SqlExecMacro(s.end(this), macro, params);
+    }
+}
+
+void SqlExecMacroArgument(SqlNodeList params) :
+{
+    SqlNode e;
+}
+{
+    <LPAREN>
+    (
+        e = SqlSimpleIdentifierEqualLiteral()
+        {
+            params.add(e);
+        }
+        (
+            <COMMA>
+            e = SqlSimpleIdentifierEqualLiteral()
+            {
+                params.add(e);
+            }
+        )*
+    |
+        e = Literal()
+        {
+            params.add(new SqlExecMacroParam(getPos(), e));
+        }
+        (
+            <COMMA>
+            e = Literal()
+            {
+                params.add(new SqlExecMacroParam(getPos(), e));
+            }
+        )*
+    )
+    <RPAREN>
+}
+
+SqlNode SqlSimpleIdentifierEqualLiteral() :
+{
+    SqlNode name;
+    SqlNode value;
```

Comment:
Let's make this the more specific SqlLiteral type.

Reply:
Since [Literal() function](https://github.com/googleinterns/calcite/blob/5d283ceedf9563bff1c715e1393fa15d094255f2/core/src/main/codegen/templates/Parser.jj#L4207) in Parser.jj returns SqlNode, it would be reasonable to use SqlNode here. Unless we want to do type casting to make it more specific.

Reply:
Looks like you're right. SqlNode seems like the wrong return value for the Literal() function. But since that's not introduced in this change, I think it's fine to leave as is.